tools/hangar/package.json

```diff
@@ -12,13 +12,14 @@
   "author": "Monada",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@cdktf/provider-aws": "^12.0.13",
+    "@fig/complete-commander": "^2.0.1",
```

Comment:
I added this because [here](https://github.com/winglang/wing/actions/runs/5135713161/jobs/9242301503) the test failed with `Error: Cannot find module '@fig/complete-commander'`